docs/install/install.rst

```diff
@@ -4,22 +4,36 @@
 Installation
 ############
 
-=======================================
-Installing POSYDON v1.0.0 from Anaconda
-=======================================
-
-We recommend to install the Python distribution Anaconda and to install POSYDON 
-in a virtual environment. Specifically, we recommend using the installation we 
-have provided which can be accessed through conda-forge. On Linux, the new 
-conda environment can be created (we have named our environment posydon-example, 
-but you can choose any name), the conda-forge channel added, and the required 
+The easiest way to start with POSYDON is to install it from Anaconda
+(Automatic Installation), and call an executable script that downloads the grid
+data automatically (Automatic Data Download).
+
+Alternatively, if you are interested in modifying the code, you can install
+POSYDON manually through `github` (Manual Installation). You can still use the
+automatic script to download the data, however, if you are about to modify the
+grid data, you can get them using the `git-lfs` module (Manual Data Download).
```

Comment:
Is the git-lfs still in place? I remember Simone told me once, this isn't available anymore.